move2kube/specs/notifications.yaml

```diff
@@ -238,6 +231,24 @@ components:
           type: string
         message:
           type: string
+        actions:
```

Comment:
this will have to wait to one of two: PR to the generator or Pr to the notification plugin, right?

Reply:
I see it on the main of janus idp (https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/src/openapi.yaml) is it not already merged? There are so many projects to propagate a change in a given project, I am totally lost as to where it should be merged to be available to us (the showcase I guess?)